content/blog/announcing-crossguard-preview/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Announcing Crossguard Preview"
+date: 2019-12-02
+meta_desc: "Today we are announcing Pulumi CrossGuard, a Policy as Code solution, is available all Team Pro and Enterprise customers to preview."
+meta_image: meta.png
+authors: ["erin-krengel"]
+tags: ["crossguard", "policy-as-code", "New-Features", "Pulumi-News","Features"]
+---
+
+Over the past few months, we have been hard at work on Pulumi CrossGuard, a Policy as Code solution. Using the `pulumi-policy` library, you can express flexible business and security rules using code. CrossGuard enables organization administrators to enforce these policies across their organization or just on specific stacks.
+
+<!--more-->
+
+You can run policies against any Pulumi stack, independent of the stack's language. This means you can have policies written in TypeScript and run them against a Python stack. Policies are grouped in Policy Packs and run against a stack during both `pulumi preview` and `pulumi up`. Each resource in a stack is run through the Policy Pack prior to creation, meaning you can block the creation of out-of-compliance resources.
+
+## Why
+
+Using Pulumi, developers and operators are empowered to self provision their infrastructure and move quickly. One our core goals is to allow you to codify best practices and share them with your team. We often hear from organization leaders that they're concerned about how to make sure their team doesn't accidentally go off the rails. CrossGuard provides guardrails your leadership needs to allow your team to continue to operate self-sufficiently.
```

Comment:
"doesn't accidentally go off the rails" is a little vague. Perhaps "... organization leaders that they are concerned about enforce security and compliance best-practices. CrossGuard provides a way to automatically enforce compliance, allowing teams to operate self-sufficiently while still ensuring organization-wide requirements are met."